### ISSUE TITLE
Issue with scroll performance after appearing above system elements.

### ISSUE THREAD

**Issue by SonnyBlack** [nicklockwood/iCarousel#592]
Good day Nick! I have strange issue using iCarousel. I have about only 10 elements, create it and it works fine. (Smooth scroll). But sometimes I get UIAlertView above iCarousel (for example - App lost enternet connection or some another notification for user). And after that alert, iCarousel performance downs on scroll. Can you please give me advice what can happen? Thanks

**Comment by nicklockwood** [comment]
Perhaps when the carousel enters background and re-enters foreground you are creating a new copy of the carousel on top of the old one?

That might happen if you are creating your carousel view inside viewDidAppear, or something like that.

**Comment by SonnyBlack** [comment]
Hello again! I check your idea, but no - I create carousel only one time. But it seems like I fixed this issue. 
In my case Carousel element lays on SKView. This view contain another element - SKScene. I noticed that its issue also appear after performing some actions in SKScene. So I tried disable in some properties of SKView - in result after disable "paused" - Carousel smooth scrolls even after UIAlertView. Thanks for your help!
